.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
_nit_: You can find a template gitignore file for python projects and reuse it, it might be useful

Reply:
Why should I add so much extra information here? I don`t even use any toolboxes now.